### DISCUSSION TITLE
Redesign of UI and UI elements

### DISCUSSION THREAD

**Discussion by kcramakrishna** [Digital-Assistant/Digital_Assistant_Client#454] [Show and tell]
This thread is to discuss new deault UI theme.

**Comment by kcramakrishna** [comment]
Below is Received from Moen:

1. Gradient for every card looks over done, I think we can have this for hover only card if you need to retain gradient
2. Record button has color, the black was not communicative enough, shall we revert to neutral grey?
3. The search bar should be a responsive design if the record is not available then we need to make sure the search stretches end to end

 

CTA on hover

https://www.figma.com/proto/1atBhbZ6ZcPUZm6KBW2bgL/UDAN?page-id=0%3A1&type=design&node-id=1-3&viewport=733%2C592%2C0.5&t=IovA3njoG5JCcirD-8&scaling=scale-down-width&hide-ui=1

 

CTA always visible

https://www.figma.com/proto/1atBhbZ6ZcPUZm6KBW2bgL/UDAN?page-id=0%3A1&type=design&node-id=55-162&viewport=733%2C592%2C0.5&t=IovA3njoG5JCcirD-8&scaling=min-zoom&hide-ui=1

**Comment by kcramakrishna** [comment]
1. Agree with Gradient.
2. I am OK with "Record" button having color. But isnt this button too minimalistic ?i.e. its not obvious that this is 'Record'.
3. Search bar can be responsive. In fact the whole layout and EVERY element should be responsive. BUT, is it better to move the 'record' button to the left of '?' button on top right. That way, the no positions are changed.
4. Regarding the 'delete' button - that too is visible only to the person who created the sequence or to an admin. It may be better to move it on line higher and right justify it to make it in line with the Title of the sequence. You can right justify the play and view icons ?

**Reply by Moen** [reply]
1. Alright will make gradients only for hover.
2. It is the standard icon and everyone is familiar with it, changing this decades-old record icon might not be a good move, let's pair tool tips so users can understand when they hover.
3. OK have moved the record to the top right, if you think this is not a crowding all the call to actions in one tiny area then we can continue this position.
4. The title can be lengthy sometimes and it will be ideal if we reserve the right side of the title for the text and do not have any overlapping icons when the title grows, the bottom row will never grow as the text is fixed so the icons on this line are safe

Demo
https://www.figma.com/proto/1atBhbZ6ZcPUZm6KBW2bgL/UDAN?page-id=0%3A1&type=design&node-id=69-2&viewport=129%2C738%2C1&t=jQhH9y3vzA1lO3Pt-8&scaling=scale-down-width&hide-ui=1
